sdf.md

```diff
@@ -1841,28 +1856,109 @@ Specification:
 
 Repository:
 :  combining the symbol values from the {{senml-units (SenML
-   Units)<IANA.senml}} Registry and the {{secondary-units (Secondary
-   Units)<IANA.senml}} Registry in {{-units}} as specified by {{Sections
+   Units)<IANA.senml}} registry and the {{secondary-units (Secondary
+   Units)<IANA.senml}} registry in {{-units}} as specified by {{Sections
    4.5.1 and 12.1 of -senml}} and {{Section 3 of -senml-units-2}},
    respectively (which by the registration policy are guaranteed to be
    non-overlapping).
 
 Index value:
-: Percent-encoding ({{Section 2.1 of -uri}}) is required of
+: Percent-encoding ({{Section 2.1 of RFC3986@-uri}}) is required of
   any characters in unit names as required by ABNF rule "pchar" in
-  {{Section 3.3 of -uri}}, specifically at the time of writing for the
+  {{Section 3.3 of RFC3986@-uri}}, specifically at the time of writing for the
   unit names "%" (deprecated in favor of "/"), "%RH", "%EL".
 
 Registries
 ----------
 
-IANA is requested to create an "SDF Parameters" registry, with the
-sub-registries defined in this Section.
+IANA is requested to create an "SDF Parameters" registry group, with the
+registries defined in this Section.
+
+### Quality Names {#qn}
+
+IANA is requested to create a "Quality Name Prefixes" registry in
+the "SDF Parameters" registry group, with the following template:
+
+Name:
+: A name composed of ASCII letters, digits, and dollar signs, starting
```

Comment:
Maybe "quality name" instead of just "name" to make it clear? Or even "The name of the quality"? No strong opinion here though.
```suggestion
: A quality name composed of ASCII letters, digits, and dollar signs, starting
```